File: java/src/NumeroPrimo.java
import java.util.Scanner;

public class NumeroPrimo {

    static boolean verificaPrimo(int numero) {
        for (int i = 2; i < numero; i++) {
            if (numero % i == 0) {
                return false;
            }
        }
        return true;
    }

    public void numeroPrimo() {
        Scanner input = new Scanner(System.in);
        int n;

        System.out.print("Digite um numero: ");
        n = input.nextInt();

        if (verificaPrimo(n) == true) {

            System.out.println(n + " eh primo");
        } else {
            System.out.println(n + " nao eh primo");
        }
    }
}

File: javascript/NumeroPrimo.js
var input = require('fs').readFileSync('javascript/stdin', 'utf8');
//ESTUDAR
var sep = [' ', '\n'];
var line = input.split(new RegExp(sep.join('|'), 'g'));

console.log("Digite um numero:")
let primo = parseInt(line.shift());
console.log(primo)
cont = 0;
   

for(let j = 2; j < primo; j++){
    if(primo % j == 0){
        cont++;
    }

}

if(cont > 0){
    console.log(`${parseInt(primo)} nao eh primo`);
}
else{
    console.log(`${parseInt(primo)} eh primo`);
}